plugins/lookup/tss.py

```diff
@@ -294,6 +339,6 @@ def run(self, terms, variables, **kwargs):
         )
 
         try:
-            return [tss.get_secret(term) for term in terms]
+            return [tss.get_secret(term, _fetch_attachments, _file_download_path) for term in terms]
```

Comment:
Use `self.get_option()` (as above) to get hold of the values of the options.

Reply:
used self.get_option()